File: swift/Sources/HermesAPI/HermesAPI.swift
/// Entry point for the generated Hermes gateway and dashboard clients.
public enum HermesAPI {
    /// The upstream Hermes release used to generate this package.
    public static let hermesRelease = HermesGatewayContract.release
    /// The desktop gateway protocol version this package supports.
    public static let contractVersion = HermesGatewayContract.desktopContract
}

File: kotlin/src/main/kotlin/hermes/api/HermesAPI.kt
package hermes.api

import hermes.api.generated.gateway.HermesGatewayContract

/** The upstream Hermes release used to generate this package. */
public object HermesAPI {
    public const val hermesRelease: String = HermesGatewayContract.release
    public const val contractVersion: Int = HermesGatewayContract.desktopContract
}
